usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -4683,6 +4933,15 @@ CK_RV ep11tok_generate_key_pair(STDLL_TokData_t * tokdata, SESSION * sess,
                                   pPrivateKeyTemplate,
                                   ulPrivateKeyAttributeCount, sess->handle);
         break;
+    case CKM_IBM_DILITHIUM:
+        rc = dilithium_generate_keypair(tokdata, sess, pMechanism,
```

Comment:
Maybe also name the function dilithium**_ibm**_generate_keypair ?

Reply:
ok, done.